### PR TITLE
RGW Multisite Active-Paasive, Failover and Faiback

### PR DESCRIPTION
**Pull request by TejasC88**
Test suite for evaluating RGW multi-site active-passive, failover and failback scenarios
Steps:
- Setup Multisite in a active-passive format
- Do multipart uploads on primary and verify on secondary.
- Remove read-only flag from secondary and failover to secondary
- Perform regular object uploads.
- Realm pull on former primary and failback to it.
- Perform regular object uploads and verify on secondary

Pass log : http://magna002.ceph.redhat.com/ceph-qe-logs/tejas/cephci-run-ML1I9Y/

### CODE REVIEW

File: suites/reef/rgw/tier-2_rgw_ms_failover_failback.yaml [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+              - "radosgw-admin zone create --rgw-realm india --rgw-zonegroup shared --rgw-zone secondary --endpoints http://{node_ip:node5}:80 --access-key 21e86bce636c3aa0 --secret cf764951f1fdde5d --read-only=true"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "ceph config set client.rgw.{daemon_id:shared.sec} rgw_realm india"
+              - "ceph config set client.rgw.{daemon_id:shared.sec} rgw_zonegroup shared"
+              - "ceph config set client.rgw.{daemon_id:shared.sec} rgw_zone secondary"
+              - "ceph orch restart {service_name:shared.sec}"
+      desc: Setting up RGW multisite with secondary as read-only
+      module: exec.py
+      name: setup multisite
+      polarion-id: CEPH-10362
```
Comment (mkasturi18) [inline suites/reef/rgw/tier-2_rgw_ms_failover_failback.yaml:238]: @TejasC88 , could you please update the testcase for configuring multisite with readonly

**Review by mkasturi18 — COMMENTED** [review]
Review comments are to be addressed. Marking this PR with DNM flag

**Review by hmaheswa — COMMENTED** [review]
@TejasC88  could you copy this suite to pacific if this scenario is supported in pacific and quincy

**Comment by TejasC88** [comment]
> @TejasC88 could you copy this suite to pacific if this scenario is supported in pacific and quincy

@hmaheswa I have added the suite to pacific also.

File: suites/pacific/rgw/tier-2_rgw_ms_failover_failback.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Test suite for evaluating RGW multi-site deployment scenario.
```
Comment (hmaheswa) [inline suites/pacific/rgw/tier-2_rgw_ms_failover_failback.yaml:1]: @TejasC88 I think you deleted this file by mistake suites/reef/rgw/tier-1_rgw_multisite.yaml
you moved this file
suites/reef/rgw/tier-1_rgw_multisite.yaml → suites/pacific/rgw/tier-2_rgw_ms_failover_failback.yaml
please check..

**Review by hmaheswa — APPROVED** [review]

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**